src/main/java/uk/gov/hmcts/reform/et/syaapi/service/PdfMapperService.java

```diff
@@ -0,0 +1,325 @@
+            claimantOtherType.getClaimantPensionContribution();
+        if (YES.equals(pensionContributionYesNo)) {
+            printFields.put(PdfMapperConstants.Q6_PENSION_YES, claimantOtherType.getClaimantPensionContribution());
+            printFields.put(PdfMapperConstants.Q6_PENSION_WEEKLY,
+                claimantOtherType.getClaimantPensionWeeklyContribution());
+
+        } else {
+            printFields.put(PdfMapperConstants.Q6_PENSION_NO, YES);
+        }
+        printFields.put(PdfMapperConstants.Q6_OTHER_BENEFITS, claimantOtherType.getClaimantBenefitsDetail());
+        return printFields;
+    }
+
+    private Map<String, String> printRepresentative(RepresentedTypeC representativeClaimantType) {
+        if (representativeClaimantType != null) {
+            Map<String, String> printFields = createHashMap();
+            printFields.put(PdfMapperConstants.Q11_REP_NAME, representativeClaimantType.getNameOfRepresentative());
+            printFields.put(PdfMapperConstants.Q11_REP_ORG, representativeClaimantType.getNameOfOrganisation());
+            printFields.put(PdfMapperConstants.Q11_REP_NUMBER, "");
+            Address repAddress = representativeClaimantType.getRepresentativeAddress();
+            printFields.put(String.format(PdfMapperConstants.QX_HOUSE_NUMBER, REP_ADDRESS_PREFIX),
+                repAddress.getAddressLine1());
+            printFields.put(String.format(PdfMapperConstants.QX_STREET, REP_ADDRESS_PREFIX),
+                repAddress.getAddressLine2());
+            printFields.put(String.format(PdfMapperConstants.QX_POST_TOWN, REP_ADDRESS_PREFIX),
+                repAddress.getPostTown());
+            printFields.put(String.format(PdfMapperConstants.QX_COUNTY, REP_ADDRESS_PREFIX),
+                repAddress.getCounty());
+            printFields.put(String.format(PdfMapperConstants.QX_POSTCODE, REP_ADDRESS_PREFIX),
+                repAddress.getPostCode());
+            printFields.put(String.format(PdfMapperConstants.QX_PHONE_NUMBER, REP_ADDRESS_PREFIX),
+                representativeClaimantType.getRepresentativePhoneNumber());
+            printFields.put(PdfMapperConstants.Q11_MOBILE_NUMBER,
+                representativeClaimantType.getRepresentativeMobileNumber());
+            printFields.put(PdfMapperConstants.Q11_EMAIL, representativeClaimantType.getRepresentativeEmailAddress());
+            printFields.put(PdfMapperConstants.Q11_REFERENCE, representativeClaimantType.getRepresentativeReference());
+            String representativePreference = representativeClaimantType.getRepresentativePreference();
+            if ("Email".equals(representativePreference)) {
+                printFields.put(PdfMapperConstants.Q11_CONTACT_EMAIL, YES);
+            } else {
+                printFields.put(PdfMapperConstants.Q11_CONTACT_POST, YES);
+            }
+            return printFields;
+        }
+        return new HashMap<>();
+    }
+
+    private Map<String, String> createHashMap() {
```

Comment:
I'm not a great fan of this. Is it really needed? Even if it is then it doesn't do what it says on the tin. The method is createHashMap but it returns a Map. If it creates a HashMap then you ought to return a HashMap

Reply:
Do you know what the PMD warning was? It might be worth reviewing to see if we should remove it.
@nickwest1 FYI

Reply:
"Since Java5 brought a new implementation of the Map designed for multi-threaded access, you can perform efficient map reads without blocking other threads."

This is the description, changed all my maps to concurrent hashmaps

Reply:
I think you were correct originally with HashMap. But I will defer to Nick's better judgment.